tests/test_config.py

```diff
@@ -21,6 +21,24 @@ def test_lock_timeout(self):
     def test_key_prefix_default(self):
         self.assertEqual(self.conf.key_prefix, 'redbeat:')
 
+    def test_lock_key_default(self):
+        self.assertTrue("REDBEAT_LOCK_KEY" not in self.app.conf.keys())
+        self.assertTrue("redbeat_lock_key" not in self.app.conf.keys())
+        self.conf = RedBeatConfig(self.app)
+        self.assertEqual(self.conf.lock_key, 'redbeat::lock')
+
+    def test_disable_lock_key_4(self):
+        self.app.conf.redbeat_lock_key = None
+        self.assertTrue("redbeat_lock_key" in self.app.conf.keys())
+        self.conf = RedBeatConfig(self.app)
+        self.assertEqual(self.conf.lock_key, None)
+
```

Comment:
Should there be a  ```@pytest.mark.skipif(CELERY_4_OR_GREATER, reason="requires Celery < 4.x")``` here?

Reply:
Yup.  Will add the skipif

Reply:
I've dropped celery 3 support so this maybe simpler to fix.